guides/vsf1_instance.md

```diff
@@ -0,0 +1,628 @@
+# Adding XDN to Vue Storefront Instance guide
+
+## 1. Install Vue Storefront
+
+Install VSF application using this guide: https://docs.vuestorefront.io/guide/installation/linux-mac.html
+
+<b>Note</b>: XDN requires Node version >= 12, so before the installation it's recommended to run:
+```bash
+nvm use 12
+```
+
+## 2. Prepare VSF files for XDN
+
+- In the new VSF project, go to `src/themes/default` (or any theme you're using) and remove `.git` folder from it to save that in Git VCS.
+- Go to `.gitignore` file and remove `config/local.json` line to keep it tracked.
+
+## 3. Install XDN
+
+- Install XDN packages: `yarn add -D @xdn/cli && yarn add @xdn/core @xdn/prefetch @xdn/devtools`
```

Comment:
Shouldn't we suggest using `xdn init` instead of installing the individual packages/files manually?

Reply:
I think that's a kind of custom case where we need to configure that manually

Reply:
Perhaps there is a bug in the CLI since ideally this should work. Trying it myself I get the following error:
```
vue-storefront git:(master) ✗ xdn init
ℹ installing @xdn/core, @xdn/cli, @xdn/prefetch, @xdn/devtools...
yarn add v1.19.1
error Running this command will add the dependency to the workspace root rather than the workspace itself, which might not be what you want - if you really meant it, make it explicit by running this command again with the -W flag (or --ignore-workspace-root-check).
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
✖ installing @xdn/core, @xdn/cli, @xdn/prefetch, @xdn/devtools... failed.
Error: Process exited with code 1
    at ChildProcess.<anonymous> (/Users/tristanlee/.nvm/versions/node/v12.18.4/lib/node_modules/@xdn/cli/utils/run.js:30:16)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
Error: Process exited with code 1
    at ChildProcess.<anonymous> (/Users/tristanlee/.nvm/versions/node/v12.18.4/lib/node_modules/@xdn/cli/utils/run.js:30:16)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
```

Reply:
Actually, even running the package install as you have documented I still get the error:
```
vue-storefront git:(master) ✗ yarn add -D @xdn/cli && yarn add @xdn/core @xdn/prefetch @xdn/devtools
yarn add v1.19.1
error Running this command will add the dependency to the workspace root rather than the workspace itself, which might not be what you want - if you really meant it, make it explicit by running this command again with the -W flag (or --ignore-workspace-root-check).
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

If this is the correct place to install, I think we need to document the `-W` flag as well.

Reply:
Oh, yea, good catch! It's a `-W` flag should be added

Reply:
just added